src/main/resources/application.properties

```diff
@@ -1,10 +1,20 @@
+#spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
probably this file should be the part of hibernate task

Reply:
improved with commit a26ad1d